### DISCUSSION TITLE
Moving git2nixsha.dev:1506 to a standard HTTP/HTTPS port

### DISCUSSION THREAD

**Discussion by frafra** [ropensci/rix#281] [General]
Hi :)
Various research institutions have a rather strict network policy, and they do not allow connecting to non-standard ports. It would be great to have git2nixsha.dev:1506 served on port 80 or 443.

Thank you :)

**Comment by b-rodrigues** [comment]
Hi, thanks I’ll take a look at this, shouldn’t be too hard :)

**Comment by b-rodrigues** [comment]
I've updated the api to serve from port 80 with the latest commit. Could you give it a try?

**Comment by b-rodrigues** [comment]
It's now being served from port 443 with https

**Reply by frafra** [reply]
It works well, thank you :)
